datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -177,59 +177,51 @@ impl LogicalPlan {
     /// Get a vector of references to all schemas in every node of the logical plan
```

Comment:
I think we should update the docstring of this function to reflect its new purpose